.changeset/little-gifts-do.md

```diff
@@ -0,0 +1,7 @@
+---
+"@rocket.chat/meteor": minor
+"@rocket.chat/i18n": minor
+"@rocket.chat/license": minor
+---
+
+Introduced restrictions (connect to cloud or have a license) to air-gapped environments
```

Comment:
We should use the new changeset guidelines, so the verb here should be on present tense